python/tests/test_3rd_harm_1d.py

```diff
@@ -53,7 +52,8 @@ def test_3rd_harm_1d(self):
 
         harmonics = [self.k, self.amp, mp.get_fluxes(self.trans1)[0], mp.get_fluxes(self.trans3)[0]]
 
-        np.testing.assert_allclose(expected_harmonics, harmonics)
+        tol = 3e-5 if mp.is_single_precision() else 1e-7
```

Comment:
This also makes sense — there is little hope of getting 7 significant digits in single precision.